libraries/AP_HAL_Linux/HAL_Linux_Class.cpp

```diff
@@ -10,6 +10,7 @@
 
 #include "AP_HAL_Linux_Private.h"
 #include "HAL_Linux_Class.h"
+#include <AP_Module/AP_Module.h>
```

Comment:
nitpick: on Linux hal we try to maintain the headers sorted with:  system, libraries, local headers.